packages/web/components/Form/FormWizard.js

```diff
@@ -142,6 +143,9 @@ const getForm = (steps, currentStep) => {
 	return currentStep !== '' && findStep ? findStep.form : steps[0].form;
 };
 
+const parseCostValueToInt = (costs = []) =>
+	costs.map((cost) => ({ ...cost, value: formatCurrencyToInt(cost.value) }));
```

Comment:
This was only suggested to improve readability👇 
```suggestion
const parseCostValueToInt = (costs = []) => {
	return costs.map((cost) => ({
		...cost,
		value: formatCurrencyToInt(cost.value),
	}));
};
```

Reply:
Done 🤝

Reply:
It's only a implicit return, less code to write 😄

Reply:
BTW, I had to do this way to safeguard the code as @lcnogueira pointed